commit fd13b5afa162b756a1dffb437b6ac618363d04ea
Author: Kieran Bingham <kieran.bingham@ideasonboard.com>
Date:   Mon Mar 18 17:17:40 2019 +0000

    vcsm: Reduce scope of local functions
    
    The functions:
    
      vc_vchi_sm_send_msg
      vc_sm_ioctl_alloc
      vc_sm_ioctl_alloc_share
      vc_sm_ioctl_import_dmabuf
    
    Are declared without a prototype. They are not used outside of this
    module, thus - convert them to static functions.
    
    Signed-off-by: Kieran Bingham <kieran.bingham@ideasonboard.com>

diff --git a/drivers/char/broadcom/vc_sm/vc_vchi_sm.c b/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
index 673379ab954a..fda7e378652e 100644
--- a/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
+++ b/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
@@ -361,7 +361,7 @@ int vc_vchi_sm_stop(struct sm_instance **handle)
 	return -EINVAL;
 }
 
-int vc_vchi_sm_send_msg(struct sm_instance *handle,
+static int vc_vchi_sm_send_msg(struct sm_instance *handle,
 			enum vc_sm_msg_type msg_id,
 			void *msg, uint32_t msg_size,
 			void *result, uint32_t result_size,
diff --git a/drivers/char/broadcom/vc_sm/vmcs_sm.c b/drivers/char/broadcom/vc_sm/vmcs_sm.c
index 38c4e17b39a6..c3462d65e1f7 100644
--- a/drivers/char/broadcom/vc_sm/vmcs_sm.c
+++ b/drivers/char/broadcom/vc_sm/vmcs_sm.c
@@ -1558,8 +1558,8 @@ static int vc_sm_mmap(struct file *file, struct vm_area_struct *vma)
 }
 
 /* Allocate a shared memory handle and block. */
-int vc_sm_ioctl_alloc(struct sm_priv_data_t *private,
-		      struct vmcs_sm_ioctl_alloc *ioparam)
+static int vc_sm_ioctl_alloc(struct sm_priv_data_t *private,
+			     struct vmcs_sm_ioctl_alloc *ioparam)
 {
 	int ret = 0;
 	int status;
@@ -1669,8 +1669,8 @@ int vc_sm_ioctl_alloc(struct sm_priv_data_t *private,
 }
 
 /* Share an allocate memory handle and block.*/
-int vc_sm_ioctl_alloc_share(struct sm_priv_data_t *private,
-			    struct vmcs_sm_ioctl_alloc_share *ioparam)
+static int vc_sm_ioctl_alloc_share(struct sm_priv_data_t *private,
+				   struct vmcs_sm_ioctl_alloc_share *ioparam)
 {
 	struct sm_resource_t *resource, *shared_resource;
 	int ret = 0;
@@ -2184,9 +2184,9 @@ static int vc_sm_ioctl_unlock(struct sm_priv_data_t *private,
 }
 
 /* Import a contiguous block of memory to be shared with VC. */
-int vc_sm_ioctl_import_dmabuf(struct sm_priv_data_t *private,
-			      struct vmcs_sm_ioctl_import_dmabuf *ioparam,
-			      struct dma_buf *src_dma_buf)
+static int vc_sm_ioctl_import_dmabuf(struct sm_priv_data_t *private,
+				     struct vmcs_sm_ioctl_import_dmabuf *ioparam,
+				     struct dma_buf *src_dma_buf)
 {
 	int ret = 0;
 	int status;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1195_fd13b5afa162b756a1dffb437b6ac618363d04ea.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1195_fd13b5afa162b756a1dffb437b6ac618363d04ea.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
